### PR TITLE
Update README.md, erased old version desc

### PR DESCRIPTION
**Pull request by ekorolevich**
Deleted 4x-version folder
Placed README.md to the root folder
Changed repo to marketplace.gcr.io in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,400 @@
...
+# <a name="using-kubernetes"></a>Using Kubernetes
+
+Consult [Launcher container documentation](https://cloud.google.com/launcher/docs/launcher-container)
+for additional information about setting up your Kubernetes environment.
+
+## <a name="running-wordpress-kubernetes"></a>Running Wordpress
+
+This section describes how to spin up a Wordpress service using this image.
+
+### <a name="run-wordpress-and-mysql-containers-kubernetes"></a>Run Wordpress and MySQL containers
```
Comment (ovk6) [inline README.md:44]: Wordpress -> WordPress here and everywhere

File: README.md [hunk 2]
```diff
@@ -1,13 +1,400 @@
...
-============
+# <a name="about"></a>About
 
-Dockerfile source for WordPress [docker](https://docker.io) image.
+This image contains an installation of WordPress 5.x served by an Apache HTTP
+Server on a PHP 7.x runtime.
 
-# Upstream
+For more information, see the
+[Official Image Launcher Page](https://console.cloud.google.com/launcher/details/google/wordpress5-php7-apache).
```
Comment (ovk6) [inline README.md:7]: ```[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/details/google/wordpress5-php7-apache).```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,400 @@
...
-https://github.com/docker-library/wordpress
+Pull command (first install [gcloud](https://cloud.google.com/sdk/downloads)):
 
-# Disclaimer
+```shell
+gcloud docker -- pull marketplace.gcr.io/google/wordpress5-php7-apache
+```
 
-This is not an official Google product.
+Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlatform/wordpress-docker/tree/master/5/php7/debian9/apache).
```
Comment (ovk6) [inline README.md:15]: Remove this link, it's redundant in this case

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,13 +1,400 @@
-wordpress-docker
-============
+# <a name="about"></a>About
```
[the remark this answers is not in the document]
Comment (ekorolevich) [reply README.md:15]: Done

File: README.md (hunk 2 above)
Comment (ekorolevich) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (ekorolevich) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (ekorolevich) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,13 +1,400 @@
...
 
-Dockerfile source for WordPress [docker](https://docker.io) image.
+This image contains an installation of WordPress 5.x served by an Apache HTTP
```
[the remark this answers is not in the document]
Comment (ekorolevich) [reply README.md]: Done

File: versions.yaml [hunk 6]
```diff
@@ -56,26 +56,6 @@ versions:
     - tests/functional_tests/running_test.yaml
     - tests/functional_tests/apache_exporter_test.yaml
 
-  # WordPress 4 with PHP 7.3-apache on Debian 9
```
Comment (ovk6) [inline versions.yaml:59]: WordPress 5 with PHP 7.3-fpm on Debian 9 section can also be deleted with corresponding folder

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 6 above)
Comment (ekorolevich) [reply versions.yaml:59]: Deleted #WordPress 5 with PHP 7.3-fpm on Debian 9 section in vesrions.yaml and deleted 5/php7/debian9/fpm directory

**Review by ovk6 — APPROVED** [review]

**Comment by ekorolevich** [comment]
@googlebot I signed it!

**Pull request merged**